### DISCUSSION TITLE
v0.3.1

### DISCUSSION THREAD

**Discussion by mkckr0** [mkckr0/audio-share#114] [Announcements]
Android App
- Change the default host to `192.168.`
- Add click media notification to start Activity
- Fix the playlist exists when player stop
- Fix the QS Tile can't stop playback service
- Add stop button to media notification

Server MFC
- Fix DPI problem

<hr /><em>This discussion was created from the release <a href='https://github.com/mkckr0/audio-share/releases/tag/v0.3.1'>v0.3.1</a>.</em>

**Comment by AlphaBack** [comment]
The v0.3.1 apk has no sound on both my mobile phones. It works fine after returning the v0.3.0 apk.
v0.3.1 apk 在我两个手机上都没声音，退回v0.3.0 apk 才正常。

**Reply by mkckr0** [reply]
显示“Started”之后没声音吗？有server.log吗？

**Reply by AlphaBack** [reply]
有的，显示已连接上电脑端的了。你测试正常的吗？

**Reply by mkckr0** [reply]
每个版本公开发布之前都会测试。下载 audio-share-server-cmd-windows.zip，使用 `as-cmd -V -b` 测试可以看到更详细的日志

**Comment by AlphaBack** [comment]
[server.log](https://github.com/user-attachments/files/18267975/server.log)
![Screenshot_20241229_07172722](https://github.com/user-attachments/assets/6bfd564b-998b-40df-8982-9af2099b344f)

**Reply by AlphaBack** [reply]
应该是 Android 版本问题。两部 Android 6 手机用 v0.3.1 apk 都没声音，用v0.3.0 apk 正常。但是还有一部 Android 13 手机用 v0.3.1 apk 是正常的。电脑端是 Linux 的。

**Reply by mkckr0** [reply]
#117
